OgcApi.Net.MbTiles/MbTilesProvider.cs

```diff
@@ -112,9 +113,23 @@ public async Task<byte[]> GetTileAsync(string collectionId, int tileMatrix, int
             if (tileOptions.MaxZoom.HasValue && tileMatrix > tileOptions.MaxZoom.Value)
                 return null;
 
+            var dateString = "";
+            if (dateTime != null)
+            {
+                var dateTimeInterval = Temporal.DateTimeInterval.Parse(dateTime);
+                dateString = "_" + dateTimeInterval.Start.Value.ToString("dd-MM-yyyy");
```

Comment:
Check dateTimeInterval.Start for null before accessing value

Reply:
dateTimeInterval.Start is checked before: 
if (dateTime != null){
...
}